go.mod

```diff
@@ -34,4 +34,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220405142115-a332d4046e57
```

Comment:
Reminder.

Reply:
addressed in [a04ef52](https://github.com/codeready-toolchain/toolchain-e2e/pull/531/commits/a04ef52075a850d4405faf97bb5a3a9157aa2ee5)